source/assetlist.json

```diff
@@ -29,6 +29,53 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/source/images/source.svg"
         }
       ]
+    },
+    {
+      "description": "Source Protocol's SRCX Token Bridged from BNB Through Planq",
+      "denom_units": [
+        {
+          "denom": "ibc/FC5A7360EEED0713AE3E83E9D55A69AF873056A172AC495890ACE4582FF9685A",
+          "exponent": 0,
+          "aliases": [
+            "srcx"
+          ]
+        },
+        {
+          "denom": "srcx",
+          "exponent": 9
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/FC5A7360EEED0713AE3E83E9D55A69AF873056A172AC495890ACE4582FF9685A",
+      "name": "SRCX",
+      "display": "srcx",
+      "symbol": "SRCX",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "planq",
+            "base_denom": "srcx",
```

Comment:
Please correct `base` and `path` to use:
erc20/0x091F9A57A3F58d758b6572E9d41675918EAC7F09
(not 'srcx')

Reply:
Done. Thank you.